### PR TITLE
feat: add PointsDashboard [SF-1169]

### PR DESCRIPTION
**Pull request by biga816**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -64,6 +64,7 @@ jobs:
           NEXT_PUBLIC_USE_PACKAGE_VERSION=${{ vars.USE_PACKAGE_VERSION }}
           NEXT_PUBLIC_SUPPORTED_CHAIN_IDS=${{ vars.SUPPORTED_CHAIN_IDS }}
           NEXT_PUBLIC_NON_SUBGRAPH_SUPPORTED_CHAIN_IDS=${{ vars.NON_SUBGRAPH_SUPPORTED_CHAIN_IDS }}
+          NEXT_PUBLIC_GRAPHQL_SERVER_URL=${{ vars.GRAPHQL_SERVER_URL }}
```
Comment (as472) [inline .github/workflows/build.yml:67]: other variables like NEXT_PUBLIC_REFERRAL_MESSAGE, NEXT_PUBLIC_REFERRAL_HASHTAG?

File: src/components/atoms/Identicon/Identicon.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import * as jdenticon from 'jdenticon';
+import React, { useEffect, useRef } from 'react';
+
+interface IdenticonProps {
+    value: string;
+    size: number;
+}
+
+export const Identicon: React.FC<IdenticonProps> = ({ value, size }) => {
```
Comment (as472) [inline src/components/atoms/Identicon/Identicon.tsx:9]: @biga816 we avoid using React.FC.
export const Identicon = ({value, size} : IdenticonProps) => {}

File: .github/workflows/build.yml (hunk 1 above)
Comment (biga816) [reply .github/workflows/build.yml:67]: I added other variables.

File: src/components/atoms/Identicon/Identicon.tsx (hunk 2 above)
Comment (biga816) [reply src/components/atoms/Identicon/Identicon.tsx]: I fixed it!

**Review by leeshuzheng — DISMISSED** [review]

**Review by leeshuzheng — APPROVED** [review]

**Pull request merged**